src/pages/index.tsx

```diff
@@ -71,6 +72,20 @@ export default function Home() {
           <LandingNav />
           {showNav && <MessageComponent />}
         </div>
+        <div className="fixed bottom-8 z-40 left-1/2 -translate-x-1/2 grid place-items-center w-max">
+          <div className="absolute bg-black w-2/3 h-2/3 -z-10 rounded-full animate-ping"></div>
+          <Link href="/login"
+            className="flex items-center justify-center p-4 text-white duration-700 ease-out bg-black border-2 border-black rounded-full cursor-pointer w-fit h-fit  dark:bg-zinc-50 hover:scale-110"
+          >
+            <h4 className="flex items-center text-xl font-archivo dark:text-zinc-800 space-x-3">
+              <span className="hidden lg:block">Explore Courses</span>
+              <span className="lg:hidden">Get Started</span>
+              <span className="text-white material-icons dark:text-black">
+                north_east
+              </span>
+            </h4>
+          </Link>
+        </div>
```

Comment:
Instead of putting this directly in the `index.tsx`, I think we should create a component and then use that here. It would be more clean and maintainable in the long run.